src-api/source/includes/_builds_and_jobs.md

```diff
@@ -165,26 +165,26 @@ curl -X POST https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/
   "body" : "le bump", // commit message body
   "why" : "retry", // short string explaining the reason we built
   "dont_build" : null, // reason why we didn't build, if we didn't build
-  "queued_at" : "2013-05-24T19:37:59.095Z" // time build was queued
-  "start_time" : null, // time build started running
-  "stop_time" : null, // time build finished running
+  "queued_at" : "2013-05-24T19:37:59.095Z" // time the job was queued
+  "start_time" : null, // time the job started running
+  "stop_time" : null, // time the job finished running
   "build_time_millis" : null,
   "username" : "circleci",
   "reponame" : "mongofinil",
   "lifecycle" : "queued",
   "outcome" : "canceled",
   "status" : "canceled",
   "canceled" : true,
-  "retry_of" : 25, // build_num of the build this is a retry of
-  "previous" : { // previous build
+  "retry_of" : 25, // build_num of the job that this is a retry of
+  "previous" : { // previous job
     "status" : "success",
     "build_num" : 25
   }
 }
 ```
 
 
-**`POST` Request:** Cancels the build and then returns a summary of the build.
+**`POST` Request:** Cancels the job and then returns a summary of the job.
 
 ## Trigger a new Job
```

Comment:
I just tried this and it only works if you have a job called "build" and it only triggers the job "build"

Reply:
@rosieyohannan Oof, thank you for catching that; I tried it as well and got the same result. I'll check internally